p2p/libp2p/netMessenger.go

```diff
@@ -180,7 +178,14 @@ func createMessenger(
 
 	go func(pubsub *pubsub.PubSub, plb p2p.ChannelLoadBalancer) {
 		for {
-			netMes.sendMessage()
+			sendableData := plb.CollectOneElementFromChannels()
+
+			if sendableData == nil {
+				continue
+			}
+
+			_ = pb.Publish(sendableData.Topic, sendableData.Buff)
```

Comment:
In the previous "sendMessage" method the error was handled with a log.Trace. Don't we still want it?

``
err := topic.Publish(context.Background(), sendableData.Buff)
	if err != nil {
		log.Trace("error sending data",
			"error", err)
	}
``

Reply:
added trace

Reply:
ok